finger_to_geom_name_map:
  thumb: leap_hand_right::thumb_fingertip_collision
  index: leap_hand_right::fingertip_collision
  middle: leap_hand_right::fingertip_2_collision
  ring: leap_hand_right::fingertip_3_collision

object_dofs_dict:
  task_spec: 'leap_3d_zrot_any_cylinder'
  xyz: []     #
  rpy: ['rz']    # rz

joint_names: ['joint_1', 'joint_0', 'joint_2', 'joint_3', 'joint_12', 'joint_13', 'joint_14', 'joint_15', 'joint_5', 'joint_4', 'joint_6', 'joint_7', 'joint_9', 'joint_8', 'joint_10', 'joint_11']
